docs/compass/03-01-security.md

```diff
@@ -38,22 +32,22 @@ The table is used by Director and Tenant Mapping Handler. It contains the follow
 - `app_id` foreign key of type UUID
 - `runtime_id` foreign key of type UUID
 - `integration_system_id` foreign key of type UUID
-- `value` of type JSONB (with authentication details, such as `client_id/client_secret` in OAuth 2.0 authentication flow, `username/password` in Basic authentication flow; in case of certificates flow it is empty)
+- `value` of type JSONB (with authentication details, such as `client_id/client_secret` in OAuth 2.0 authentication flow, `username/password` in the basic authentication flow; in case of certificates flow it is empty)
```

Comment:
```suggestion
- `value` of type JSONB (with authentication details, such as `client_id/client_secret` in OAuth 2.0 authentication flow, `username/password` in the Basic authentication flow; in case of certificates flow it is empty)
```